### ISSUE TITLE
Android Firebase analytics log successful/failed events to logcat (without terminal commands)

### ISSUE THREAD

**Issue by slikasgiedrius** [firebase/firebase-android-sdk#3494]
I have a use case of logging successful/failed events inside Android logcat. Every single documentation page I have seen is about going to the console or launching commands inside terminal, but I need a native solution to display them and make pop up messages in result of a successfull event which I have created. 

Any help is really appreciated, thanks in advance.

**Comment by argzdev** [comment]
Thanks for reporting, @slikasgiedrius. Just to clarify, by native solution to display analytics data - do you mean you want to display the data that is being tracked on the Firebase analytics dashboard within your application? If it is, as of the moment, there's no native solution for that.

However, as an alternative solution, perhaps you could try creating a [realtime report](https://developers.google.com/analytics/devguides/reporting/data/v1/realtime-basics#http), and then use the data you receive from there to display it in your application.

I've also marked this as a feature request, so our engineers can take a look once they have the time.

**Comment by slikasgiedrius** [comment]
@argzdev Not really, I want to be able to display a message for internal app users when a certain `firebaseAnalytics.logEvent` event is being called and whether it's successfully logged or not.

The toast message itself would just display event name and it's parameters.

**Comment by argzdev** [comment]
Thanks for clarifying. So you want a listener when events are being logged, this way you can add logic when an event is successfully logged or not. 

I've marked this as a feature request, so that our engineers can take a look at this once they have the time.

**Comment by jimlyas** [comment]
Events are not automatically sent to Analytics when they are logged, but rather they are bundled together and being sent each hour. 

So unless you're using debug view, you can't actually track if your event is logged or not in real-time.

**Comment by slikasgiedrius** [comment]
So then I will rephrase - Is there a way to detect when event is successfully bundled?

**Comment by RahulSDeshpande** [comment]
@slikasgiedrius Any update??

**Comment by argzdev** [comment]
Yeah, you're right @jimlyas! Thanks for adding those details!

@slikasgiedrius so currently there's no way to detect when the events are successfully bundled and sent to the server. Given with this scenario that events are only sent every hour. How would you want your feature request to ideally work? Would it still help your use case to see toast messages even if they are delayed for an hour?

Hi @RahulSDeshpande, thank you for taking an interest in this feature request. You could help us track the demand on this feature by adding a thumbs up emoji on the original author's post. This will help us put feature request's in our radar depending on demand.
